packages/near-membrane-base/src/membrane.ts

```diff
@@ -991,7 +1128,7 @@ export function createMembraneMarshall() {
                     return localValue;
                 }
                 if (key === TO_STRING_TAG_SYMBOL) {
-                    return foreignCallableGetUnbrandedTag(foreignTargetPointer);
+                    return this.unbrandedTag;
```

Comment:
this seems to be a installed getter into the handler, which seems also very odd.

Reply:
The getter allows for a frozen `BoundaryProxyHandler`. We could instead have a `state` property that is bolded on the `BoundaryProxyHandler` that is mutable despite the handler being frozen.